.github/gh-actions-self-hosted-runners/arc/environments/beam.env

```diff
@@ -30,9 +28,33 @@ github_app_id_secret_name = "gh-app_id"
 github_app_install_id_secret_name = "gh-app_installation_id"
 github_private_key_secret_name = "gh-pem_key"
 deploy_webhook = "true"
-max_main_replicas = "50"
-min_main_replicas = "5"
-webhook_scaling = "true"
 runner_group = "beam"
-machine_type = "e2-standard-16"
-#state_bucket_name = "beam-arc-state"
\ No newline at end of file
+main_runner = {
+    name = "main-runner"
+    machine_type = "e2-standard-16"
+    min_node_count = "1"
+    max_node_count = "7"
+    min_replicas = "1"
+    max_replicas = "45"
+    webhook_scaling = true
+    disk_size_gb = 200
+    requests = {
+        cpu = "2"
+        memory = "3Gi"
+    }
+}
+additional_runner_pools = [{
+    name = "small-runner"
+    machine_type = "e2-standard-2"
+    min_node_count = "1"
+    max_node_count = "10"
+    min_replicas = "1"
+    max_replicas = "10"
+    webhook_scaling = "true"
+    requests = {
+        cpu = 1500m
```

Comment:
it make sense to also specify the memory value

Reply:
At this point no. We use cpu to achieve 1 per node so memory is not important in this context.